packages/flutter/lib/src/gestures/tap.dart

```diff
@@ -178,6 +222,15 @@ abstract class BaseTapGestureRecognizer extends PrimaryPointerGestureRecognizer
   @protected
   void handleTapUp({required PointerDownEvent down, required PointerUpEvent up});
 
+  /// A pointer that triggered a tap has moved.
+  ///
+  /// This triggers on the move event if the recognizer has recognized the tap gesture.
+  ///
+  /// The parameter `move` is the move event of the primary pointer that started
+  /// the tap sequence.
+  @protected
+  void handleTapMove({required PointerMoveEvent move});
```

Comment:
Could this possibly be a breaking change for users that implement BaseTapGestureRecognizer? Would it be desirable to add a default implementation here to avoid this, or do we want to break people that implement BaseTapGestureRecognizer?

Reply:
I'm running the Google tests on this PR to check if there are breakages there.

Reply:
Recently there has been [a complaint](https://github.com/flutter/flutter/pull/155575#issuecomment-2637011345) about we adding such methods breaking packages that implement it. I think it'll help developers if we mention such "non-breaking-by-Flutter-standards" changes, especially "notable new methods" in the patchnotes. What do you think?

Reply:
Yeah that sounds like a good idea to me. I've seen this kind of thing come up before too.

I think in this case it makes sense to allow users to subtype BaseTapGestureRecognizer, right? And if they are, then do they care about implementing handleTapMove? Maybe not, since I guess they would probably have some specific use case for their subclass that doesn't care about this "tap and move" bug.

Reply:
Yeah it makes sense to provide a default implementation in this case too.